validator/src/bootstrap.rs

```diff
@@ -503,13 +515,25 @@ fn get_vetted_rpc_nodes(
                         Duration::from_secs(5),
                     );
 
-                    Some((rpc_contact_info, snapshot_hash, rpc_client))
+                    let ping_time = ping(&rpc_contact_info.gossip().unwrap()); 
```

Comment:
Could copy the style above to make sure we don't panic in the case gossip address is invalid:
`rpc_contact_info.gossip().ok()?`